### PR TITLE
Added U32 precision

### PR DESCRIPTION
**Pull request by ilyachur**

### CODE REVIEW

File: inference-engine/src/inference_engine/blob_transform.cpp [hunk 1]
```diff
@@ -139,6 +140,12 @@ static inline void blob_copy_4d(Blob::Ptr src, Blob::Ptr dst) {
     case Precision::I8:
         blob_copy_4d_t<Precision::U8>(src, dst);
         break;
+    case Precision::U64:
+        blob_copy_4d_t<Precision::U64>(src, dst);
+        break;
+    case Precision::I64:
```
Comment (ilya-lavrenov) [inline inference-engine/src/inference_engine/blob_transform.cpp:146]: do we need different specialization for 64 bits data type?

File: inference-engine/tests/functional/inference_engine/blob_copy_test.cpp [hunk 2]
```diff
@@ -207,6 +211,8 @@ bool IsCorrectBlobCopy(Blob::Ptr& srcBlob, Blob::Ptr& dstBlob) {
             return IsCorrectBlobCopy_Impl<int16_t>(srcBlob, dstBlob);
         case InferenceEngine::Precision::I32:
             IsCorrectBlobCopy_Impl<int32_t>(srcBlob, dstBlob);
+        case InferenceEngine::Precision::U32:
```
Comment (ilya-lavrenov) [inline inference-engine/tests/functional/inference_engine/blob_copy_test.cpp:214]: `I32` case does not have return..

File: inference-engine/src/inference_engine/blob_transform.cpp (hunk 1 above)
Comment (ilya-lavrenov) [reply inference-engine/src/inference_engine/blob_transform.cpp:146]: BTW, do we need such specializations at all? Looks like nobody need them before.

Comment (ilyachur) [reply inference-engine/src/inference_engine/blob_transform.cpp:146]: Ok will remove

File: inference-engine/tests/functional/inference_engine/blob_copy_test.cpp (hunk 2 above)
Comment (ilyachur) [reply inference-engine/tests/functional/inference_engine/blob_copy_test.cpp:214]: Fixed

File: inference-engine/src/vpu/graph_transformer/src/frontend/frontend.cpp [hunk 3]
```diff
@@ -393,6 +393,7 @@ ModelPtr FrontEnd::runCommonPasses(ie::ICNNNetwork& network, const UnsupportedLa
         }
 
         ie::NetPass::ConvertPrecision(*originalOrConvertNetwork, ie::Precision::I64, ie::Precision::I32);
+        ie::NetPass::ConvertPrecision(*originalOrConvertNetwork, ie::Precision::U32, ie::Precision::I32);
```
Comment (gladilov-gleb) [inline inference-engine/src/vpu/graph_transformer/src/frontend/frontend.cpp:396]: Sync required logic with https://github.com/openvinotoolkit/openvino/pull/1304

Comment (ilyachur) [reply inference-engine/src/vpu/graph_transformer/src/frontend/frontend.cpp:396]: Look like these PRs can be merged separately...

Comment (gladilov-gleb) [reply inference-engine/src/vpu/graph_transformer/src/frontend/frontend.cpp:396]: What I meant is are we sure that we do U32 -> I32 conversion in an expected way?

**Pull request merged**